Cue/app/common/CueTabs.ios.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+'use strict';
+
+import React from 'react'
+import { View, Text, SegmentedControlIOS } from 'react-native'
+
+import CueColors from './CueColors'
+
+const styles = {
+  container: {
+    marginHorizontal: 16,
+    marginVertical: 12,
+  },
+}
+
+export default class CueTabs extends React.Component {
+  props: {
+    tabs: Array<string>,
+    currentTab: string,
+    onChange: (tab: number) => void,
+  }
+
+  _onChange = ({nativeEvent: {selectedSegmentIndex}}: {nativeEvent: {selectedSegmentIndex: number}}) => {
```

Comment:
Sure, it's a destructured argument, but it just looks funny because of the Flow type annotation. Equivalent to:

```js
_onChange = (event) => {
  let selectedSegmentIndex: number = event.nativeEvent.selectedSegmentIndex
  // ...
}
```